library-extension-rx/src/main/java/com/instacart/library/truetime/TrueTimeRx.java

```diff
@@ -211,7 +220,7 @@ public void subscribe(@NonNull FlowableEmitter<long[]> o)
                                               o.onNext(requestTime(singleIpHostAddress));
                                               o.onComplete();
                                           } catch (IOException e) {
-                                              o.tryOnError(e);
+                                              o.onError(e);
```

Comment:
Let it cause undeliverable exception so users can handle it, instead of crash. #101